### PR TITLE
Add guides for using the Media Creation Tool and Installing Windows 10

### PR DESCRIPTION
**Pull request by portal5555**
Also updated index.md a little bit to add links to the articles. I'm happy to oblige by any changes that may need to be made.

### CODE REVIEW

File: installing-windows-10.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Installing Windows 10
+
+
+This article covers the steps of re-installing, or fresh installing Windows 10 from a USB.
+
+## Prerequisites
+* For Method 1, the drive that you are installing Windows to requires enough space to move the contents of the drive to a folder called windows.old.
```
Comment (sylveon) [inline installing-windows-10.md]: Move this to method 1

File: installing-windows-10.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Installing Windows 10
+
+
+This article covers the steps of re-installing, or fresh installing Windows 10 from a USB.
+
+## Prerequisites
+* For Method 1, the drive that you are installing Windows to requires enough space to move the contents of the drive to a folder called windows.old.
+* The disk that you are trying to Install Windows on must be a GPT partition style disk if you are on a UEFI system, otherwise it will not let you install. 
+  * This can be worked around by booting in Legacy mode, or by formatting the disk and then converting it to GPT using Diskpart.
```
Comment (sylveon) [inline installing-windows-10.md]: Don't recommend UEFI users to install using CSM.

File: installing-windows-10.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Windows will now start re-installing on the partiton that you selected. It will move any data from that partition to a folder located at C:\windows.old if there is enough space.
+
+## Method 2
+
+
+
+> Clean re-installing, not keeping any data
+
+1. On the next screen, you will see a list of partitions for each disk you have, you want to select the partition marked "Primary" on the disk that you have Windows installed on, and that has a similar amount of space as the drive you have Windows installed on. (If Windows is installed on Disk 0, and it's maximum amount of space is 30GB, then you would select the partition on Drive 0 with 30GB of total space marked as "Primary"). After this, click the "Format" button at the bottom. **THIS WILL ERASE ALL DATA ON THE SELECTED PARTITION** and then click Next.
```
Comment (sylveon) [inline installing-windows-10.md]: Picture?

File: installing-windows-10.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+After this, click Next.
+
+Windows will now start re-installing on the partiton that you selected. It will move any data from that partition to a folder located at C:\windows.old if there is enough space.
+
+## Method 2
+
+
+
+> Clean re-installing, not keeping any data
```
Comment (sylveon) [inline installing-windows-10.md]: Might want to mention that its possible to remove all partitions on the drive and then select the unallocated space on the drive to let windows redo its partition setup. Mostly useful when it's a brand new drive (windows requires multiple partitions but the drive might come with a single partition pre made) or your partition setup is broken.

File: using-the-media-creation-tool.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# Using The Media Creation Tool
+
+
+One of the most common solutions to many problems in Windows is reinstalling it. While built-in settings exist that do this, they rely on an embedded image that  can often become unstable or unusable, and as such the Media Creation Tool is recommended for this purpose.
```
Comment (sylveon) [inline using-the-media-creation-tool.md]:
```suggestion
One of the most common solutions to many problems in Windows is reinstalling it. While built-in settings exist that do this, they rely on an embedded image that can often become unstable or unusable, and as such the Media Creation Tool is recommended for this purpose.
```

File: using-the-media-creation-tool.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+1. The first thing you will see is this screen, click Accept.
+
+    ![](img/using-the-media-creation-tool/licenseterms.png)
+
+2. The next screen you see will be this, wait for the installer to send you to the next.
+
+
+    ![](img/using-the-media-creation-tool/gettingthings.png)
+
+3. This is where you select what you want to do, if you want to **reinstall the Windows installation on this computer** click **Upgrade this PC now** and follow Method 1. If you **want to reinstall Windows on another computer** click **Create installation media** and follow Method 2.
```
Comment (sylveon) [inline using-the-media-creation-tool.md]:
```suggestion
3. This is where you select what you want to do, if you want to **repair the Windows installation on this computer** click **Upgrade this PC now** and follow Method 1. If you **want to reinstall Windows on another computer** click **Create installation media** and follow Method 2.
```

File: using-the-media-creation-tool.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+3. Select your USB that you have inserted. **ANY DATA ON THE USB WILL BE ERASED**
+
+    ![](img/using-the-media-creation-tool/selectusb.png)
+
+4. Wait for Windows 10 to finish downloading.
+
+    ![](img/using-the-media-creation-tool/downloading.png)
+
+After this, you will see a screen saying that the Flash Drive is ready. Click finish, take out the USB, and insert it into the computer that is not working.
```
Comment (sylveon) [inline using-the-media-creation-tool.md]: Eject the drive in Windows before taking out the USB

File: using-the-media-creation-tool.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+#### How to re-install Windows using a USB
+
+1. Turn on the computer that is not working, and repeatedly press either F12,F11,F10, or F9 depending on your manufacturer to enter the Boot Menu. To find out the key you need to press, search [Your PC's Manufacturer] boot menu key.
+
+2. Once open, move the cursor using the arrow keys to the name of the USB you have inserted and then press Enter.
+
+You will now boot to the Windows 10 Installer. For further guidance, see [Installing Windows 10](https://msft.chat/wiki/installing-windows-10)
```
Comment (sylveon) [inline using-the-media-creation-tool.md]:
```suggestion
You will now boot to the Windows 10 Installer. For further guidance, see [Installing Windows 10](installing-windows-10)
```

File: using-the-media-creation-tool.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+4. Wait for Windows 10 to finish downloading.
+
+    ![](img/using-the-media-creation-tool/downloading.png)
+
+After this, you will see a screen saying that the Flash Drive is ready. Click finish, take out the USB, and insert it into the computer that is not working.
+
+#### How to re-install Windows using a USB
+
+1. Turn on the computer that is not working, and repeatedly press either F12,F11,F10, or F9 depending on your manufacturer to enter the Boot Menu. To find out the key you need to press, search [Your PC's Manufacturer] boot menu key.
```
Comment (sylveon) [inline using-the-media-creation-tool.md]:
```suggestion
1. Turn on the computer that is not working, and repeatedly press either <kbd>F12</kbd>, <kbd>F11</kbd>, <kbd>F10</kbd>, or <kbd>F9</kbd> depending on your manufacturer to enter the Boot Menu. To find out the key you need to press, search on Google "[PC manufacturer] boot menu key".
```

File: installing-windows-10.md (hunk 4 above)
Comment (portal5555) [reply installing-windows-10.md]: I have mentioned it.

File: using-the-media-creation-tool.md (hunk 7 above)
Comment (portal5555) [reply using-the-media-creation-tool.md]: I have now stated this.

File: installing-windows-10.md (hunk 3 above)
Comment (portal5555) [reply installing-windows-10.md]: Added one

File: installing-windows-10.md (hunk 2 above)
Comment (portal5555) [reply installing-windows-10.md]: Removed that part.

File: installing-windows-10.md (hunk 1 above)
Comment (portal5555) [reply installing-windows-10.md]: I think I moved what you wanted, but I'm not sure so if you want me to move it again I will.

**Pull request merged**